### PR TITLE
Latest skin tone is remembered as new default

### PR DESCRIPTION
**Pull request by RicoChr**
A problem i noticed is that we now have a slight flicker when the gridview is updated due to the change of the default icon (the base emoji is never changed of course, just which one is displayed!)
Apart from that, this should be fully functional.

### CODE REVIEW

**Comment by RicoChr** [comment]
these code-style checks are ridiculous...

**Comment by vanniktech** [comment]
Kinda. If you want I can take care of them for you.

**Comment by RicoChr** [comment]
That would be nice.
Have you integrated these checks into your development environment? If so, i would like to know how.

**Comment by vanniktech** [comment]
I've got a [plugin](https://github.com/vanniktech/gradle-code-quality-tools-plugin) which lets you apply `PMD`, `Findbugs` and `Checkstyle` to any java | android project. And the configuration itself is under `code_quality_tools`. I just execute them via the terminal to see what's failing `./gradlew checkstyle findbugs pmd`.

File: emoji/src/main/java/com/vanniktech/emoji/EmojiPagerAdapter.java [hunk 1]
```diff
@@ -27,23 +30,26 @@
...
       newView = new EmojiGridView(pager.getContext()).init(listener, longListener, EmojiManager.getInstance().getCategories()[position - 1]);
+      gridViewsList.add((EmojiGridView) newView);
     }
-
     pager.addView(newView);
-
     return newView;
   }
 
+  public void notifyPagesChanged(){
```
Comment (vanniktech) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPagerAdapter.java]: this is probably causing the lag. Also it's not optimal. How about just changing the page that needs to be changed? (it's the one you're in and the recent one.

Comment (RicoChr) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPagerAdapter.java]: I couldn't find yet find out, how to get the current page so i just updated them all

Comment (vanniktech) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPagerAdapter.java]: There are methods inside the ViewPager for this.

Comment (RicoChr) [inline emoji/src/main/java/com/vanniktech/emoji/EmojiPagerAdapter.java]: I can get the index of the page but i have no clue on how to get the corresponding gridView. I think my gridViewsList is not in order...

**Comment by vanniktech** [comment]
I'll close this in favor of #157. I've taken your first commit and worked a bit around it so that we only update the ImageView and not even the entire GridView.

Thanks for your initial work.

**Comment by RicoChr** [comment]
You are welcome, this is still a very much a Challenge for me and i enjoy it :)

**Pull request closed without merging**